server/routes/home/controller.ts

```diff
@@ -0,0 +1,11 @@
+import { Request, Response } from 'express'
+import { Page } from '../../services/auditService'
+import { PageHandler } from '../../interfaces/pageHandler'
+
+export default class HomeController implements PageHandler {
+  public PAGE_NAME = Page.SEARCH_PRISONER_CONTACT_PAGE
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    res.render('pages/home/view')
+  }
```

Comment:
I was looking at renaming files from `view.njk` to something more meaningful? But I'll do this on my PR (coming soon), and see how it compares. Bit nervy that we'll end up with 100 files called `view.njk`!

Reply:
yeah same feeling 👍